charts/astronomer/templates/houston/api/houston-backend-secret.yaml

```diff
@@ -15,7 +15,7 @@ type: Opaque
 data:
   {{- if .Values.houston.backendSecretConnection }}
   # in prisma1 has hardcoded schema, so for now to migrate to prisma2 we need to hardcode it too.
-  connection: {{ (printf "postgresql://%s:%s@%s:%s/%s?schema=houston%24default" .Values.houston.backendConnection.user .Values.houston.backendConnection.pass .Values.houston.backendConnection.host (.Values.houston.backendConnection.port | toString) .Values.houston.backendConnection.db) | b64enc | quote }}
+  connection: {{ (printf "postgresql://%s:%s@%s:%s/%s?schema=houston%24default&%s" .Values.houston.backendConnection.user .Values.houston.backendConnection.pass .Values.houston.backendConnection.host (.Values.houston.backendConnection.port | toString) .Values.houston.backendConnection.db (.Values.houston.backendConnection.connectionLimit | toString)) | b64enc | quote }}
```

Comment:
unfortunately, this will not work.  we don't populate the "backendSecretConnection"